docs/quickstart.md

```diff
@@ -26,31 +36,24 @@ Its sole purpose is to run Kubernetes (k3s or RKE2), with everything controlled
 Elemental Teal is built in the [openSUSE Build Service](https://build.opensuse.org/package/show/isv:Rancher:Elemental:Stable:Teal53/node-image)
 and available through the [openSUSE Registry](http://registry.opensuse.org/isv/rancher/elemental/stable/teal53/15.4/rancher/elemental-node-image/5.3:latest)
 
-### What is the Rancher Elemental Stack ?
-
-The Elemental Stack consists of some packages on top of SLE Micro for Rancher
-
-- **elemental-toolkit** - includes a set of OS utilities to enable OS management via containers. Includes dracut modules, bootloader configuration, cloud-init style configuration services, etc.
-- **elemental-operator** - this connects to Rancher Manager and handles machineRegistration and machineInventory CRDs
-- **elemental-register** - this registers machines via machineRegistrations and installs them via elemental-cli
-- **elemental-cli** - this installs any elemental-toolkit based derivative. Basically an installer based on our A/B install and upgrade system
-- **rancher-system-agent** - runs on the installed system and gets instructions ("Plans") from Rancher Manager what to install and run on the system
-
 ## Prerequisites
 
  - A Rancher server (2.6.9) configured (server-url set)
      - To configure the Rancher server-url please check the [Rancher docs](https://rancher.com/docs/rancher/v2.6/en/admin-settings/#first-log-in)
+     - The Rancher server needs to be accessible on port 443 from all nodes that you are provisioning. (However, the requests are only one way so the nodes can be behind a NAT or other firewall)
```

Comment:
In case if Rancher can be exposed with public domain then `ngrok` project can be used.